### PR TITLE
Making the test view a CodeMirror editor as well.

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: lib/editing/editor_codemirror.dart [hunk 1]
```diff
@@ -238,6 +238,14 @@ class _CodeMirrorEditor extends Editor {
   @override
   void resize() => cm.refresh();
 
+  @override
+  bool get readOnly => cm.getReadOnly();
+
+  @override
+  set readOnly(bool ro) {
```
Comment (RedBrogdon) [inline lib/editing/editor_codemirror.dart]: nit: Can you one-line this?

File: lib/experimental/new_embed.dart [hunk 2]
```diff
@@ -205,15 +214,17 @@ class ConsoleTabView extends TabView {
...
-  const TestTabView(DElement element, this.testEditor) : super(element);
+class TestTabView extends EditorTabView {
+  TestTabView(DElement element, EditorFactory editorFactory)
+      : super(element, editorFactory) {
+        // Tests probably shouldn't change...
+        _editor.readOnly = true;
+      }
 
-  String get testMethod => testEditor.value;
+  String get testMethod => content;
```
Comment (RedBrogdon) [inline lib/experimental/new_embed.dart]: I'd be fine with just using `content` to get the test code in and out. Your call, though.

File: lib/editing/editor_codemirror.dart (hunk 1 above)
Comment (domesticmouse) [reply lib/editing/editor_codemirror.dart]: Done

File: lib/experimental/new_embed.dart (hunk 2 above)
Comment (domesticmouse) [reply lib/experimental/new_embed.dart]: Done

**Pull request merged**